### ISSUE TITLE
Line+Bar with Ordinal tooltip position misaligned on Firefox

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#10184]
The tooltip position is misaligned on Firefox, when using Line+Bar with Ordinal.
Not an issue if all series are the same type (line, area or bar) and not an issue in Chromium. 

![Screencast 2019-06-15 18 47 09](https://user-images.githubusercontent.com/1447303/59557716-1ca9b700-8f9e-11e9-80d9-590eb074f1c0.gif)

Sample code:
```
SELECT CAST('2019-06-01' AS DATE) AS Date, 1340.31 AS Daily, 1340.31 AS Accu, 1020.00 AS Goal UNION ALL
SELECT CAST('2019-06-02' AS DATE) AS Date, 498.72 AS Daily, 1839.01 AS Accu, 2040.00 AS Goal UNION ALL
SELECT CAST('2019-06-03' AS DATE) AS Date, 585.43 AS Daily, 2424.43 AS Accu, 3060.00 AS Goal UNION ALL
SELECT CAST('2019-06-04' AS DATE) AS Date, 897.09 AS Daily, 3321.52 AS Accu, 4080.00 AS Goal UNION ALL
SELECT CAST('2019-06-05' AS DATE) AS Date, 840.81 AS Daily, 4162.31 AS Accu, 5100.00 AS Goal
```
Metabase 0.32.9 with Firefox 67 on Ubuntu 19.04

**Comment by flamber** [comment]
When there's only one row, then it shows the tooltip correctly, but positions the graph to the left.
But just noticed the x-axis tick shows "2000", when using Timeseries.
Pinging @paulrosenzweig 
![Screencast 2019-06-15 20 01 29](https://user-images.githubusercontent.com/1447303/59558149-02290b00-8fa9-11e9-9c68-71431eaacb7e.gif)

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
